scripts/missions/bastok/3_2_To_the_Forsaken_Mines.lua

```diff
@@ -137,7 +133,6 @@ mission.sections =
             {
                 onTrade = function(player, npc, trade)
                     if
-                        player:checkDistance(npc) <= 1.6 and
```

Comment:
This should not be deleted, and was tested.

Reply:
I performed the trade succesfuly at arround 5 yalms of distance
Since its a repeatable mission, i can do a video as proof though, if needed.

Reply:
Possible it changed, but this was the case in the past.